okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -482,7 +483,7 @@ public void windowUpdate() throws Exception {
   public void windowUpdateWithInboundFlowControl() throws Exception {
     initTransport();
     MockStreamListener listener = new MockStreamListener();
-    clientTransport.newStream(method, new Metadata(), listener).request(1);
+    OkHttpClientStream stream = clientTransport.newStream(method, new Metadata(), listener);
```

Comment:
It seems like if you have the stream, you should be using the stream id in the checks below.  The 3 and 1 in the flaky tests are the http2 stream id I believe.